ScoutSuite/core/cli_parser.py

```diff
@@ -203,11 +203,16 @@ def _init_common_args_parser(self):
                             default=False,
                             action='store_true',
                             help='Do not automatically open the report in the browser.')
+        # TODO: This should be removed once the GCP refactoring is done
         parser.add_argument('--thread-config',
                             dest='thread_config',
                             type=int,
                             default=4,
                             help='Level of multi-threading wanted [1-5]; defaults to 4.')
+        parser.add_argument('--max-workers',
```

Comment:
Python documentation indicates:

> Changed in version 3.5: If max_workers is None or not given, it will default to the number of processors on the machine, multiplied by 5, assuming that ThreadPoolExecutor is often used to overlap I/O instead of CPU work and the number of workers should be higher than the number of workers for ProcessPoolExecutor.

Reading your code, I guess default max_workers will be set to `0` and I'm not sure of the expected behavior... What do you think about adding a `default=None`?

Reply:
If the argument is not provided by the user, the value is `None` by defaut.

![image](https://user-images.githubusercontent.com/6698529/55166428-7c35ba80-5145-11e9-9733-46582772defd.png)

Reply:
Perfect!